docs/templates-recipes.md

```diff
@@ -0,0 +1,87 @@
+Here you can find recipes for templates that you might want to copy and reuse them for your need to speed up template development. Feel free to create a Pull Request with your recipe.
+
+- [Using Schema objects to generate types for a given programming language](#using-schema-objects-to-generate-types-for-a-given-programming-language)
+- [Displaying separate lists for different channels operations](#displaying-separate-lists-for-different-channels-operations)
+
+### Using Schema objects to generate types for a given programming language
+
+* Java example where `$$schema$$.java` file looks like this. You need to provide your own implementation of `toJavaType` filter:
```

Comment:
Maybe we should add the filter for completeness, something like the following (cannot figure out how to nest codeblocks, so can't make it as a suggestion 🙄): 

* Java example for `$$schema$$.java` file with provided `toJavaType` filter:

With the `toJavaType` filter for convenience: 
```js
...
  Nunjucks.addFilter('toJavaType', jsonSchemaType => {
    switch (jsonSchemaType.toLowerCase()) {
      case 'string':
        return 'String';
      case 'integer':
        return  'Integer'
      case 'number':
        return 'Double';
      case 'boolean':
        return 'boolean';
    }
  });
...
``